l1-contracts/src/governance/libraries/ProposalLib.sol

```diff
@@ -101,28 +101,28 @@ library ProposalLib {
   /**
    * @notice	A stable state is one which cannoted be moved away from
    */
-  function isStable(DataStructures.Proposal storage self) internal view returns (bool) {
-    DataStructures.ProposalState s = self.state; // cache
+  function isStable(DataStructures.Proposal storage _self) internal view returns (bool) {
+    DataStructures.ProposalState s = _self.state; // cache
     return s == DataStructures.ProposalState.Executed || s == DataStructures.ProposalState.Dropped;
   }
 
-  function pendingThrough(DataStructures.Proposal storage self) internal view returns (Timestamp) {
-    return self.creation + self.config.votingDelay;
+  function pendingThrough(DataStructures.Proposal storage _self) internal view returns (Timestamp) {
+    return _self.creation + _self.config.votingDelay;
   }
 
-  function activeThrough(DataStructures.Proposal storage self) internal view returns (Timestamp) {
-    return ProposalLib.pendingThrough(self) + self.config.votingDuration;
+  function activeThrough(DataStructures.Proposal storage _self) internal view returns (Timestamp) {
+    return ProposalLib.pendingThrough(_self) + _self.config.votingDuration;
   }
 
-  function queuedThrough(DataStructures.Proposal storage self) internal view returns (Timestamp) {
-    return ProposalLib.activeThrough(self) + self.config.executionDelay;
+  function queuedThrough(DataStructures.Proposal storage _self) internal view returns (Timestamp) {
+    return ProposalLib.activeThrough(_self) + _self.config.executionDelay;
   }
 
-  function executableThrough(DataStructures.Proposal storage self)
+  function executableThrough(DataStructures.Proposal storage _self)
```

Comment:
storage values getting _ also seems odd to me, maybe a new prefix? - $?

Reply:
Still a param here, but I get you. Think it is easier for us to deal with it later though as it would need some extra rules for general storage as well if we want to be more consistant I guess 🤔

Reply:
https://github.com/LHerskind/solhint/commit/82c456c9a7c2c3a254d8de3a3c32722bc0fce699